mono/metadata/object-internals.h

```diff
@@ -665,7 +663,7 @@ typedef struct {
 	gboolean (*mono_above_abort_threshold) (void);
 	void (*mono_clear_abort_threshold) (void);
 	void (*mono_reraise_exception) (MonoException *ex);
-#ifdef TARGET_OSX
+#ifndef HOST_WIN32
```

Comment:
Try to remove ifdef?

Reply:
Yes, putting ifdefs in structures which are not perf critical is not very useful.